### PR TITLE
add 2080ti, 2080 into hardware.ts

### PR DESCRIPTION
**Pull request by thien**
added FP16 (half) FLOPs for the GPUs listed in title.

sources: 
- https://www.techpowerup.com/gpu-specs/geforce-rtx-2080.c3224
- https://www.techpowerup.com/gpu-specs/geforce-rtx-2080-ti.c3305
- https://www.techpowerup.com/gpu-specs/geforce-rtx-2070.c3252 (as x-reference to know which number to use).

### CODE REVIEW

**Review by julien-c — APPROVED** [review]
nice!

**Review by Vaibhavs10 — APPROVED** [review]
Thanks for adding this! 🤗 

Will wait for @pcuenca to give this an LGTM too - since he verified the numbers for other hardware on the list too!

File: packages/tasks/src/hardware.ts [hunk 1]
```diff
@@ -164,6 +164,14 @@ export const SKUS = {
 				tflops: 12.74,
 				memory: [12, 8],
 			},
+			"RTX 2080 Ti": {
+				tflops: 26.90,
```
Comment (pcuenca) [inline packages/tasks/src/hardware.ts:168]:
```suggestion
				tflops: 26.9,
```

**Review by pcuenca — APPROVED** [review]

**Comment by pcuenca** [comment]
Thanks for updating @Vaibhavs10! Fixing the lint and merging!

**Comment by pcuenca** [comment]
Thanks a lot @thien 🙌 My first GPU was a 2080 Ti in 2018, I still use it :)

**Pull request merged**